source/data-api/generated-endpoints.txt

```diff
@@ -0,0 +1,1364 @@
+for the collection.
+
+You can also set up an :ref:`IP Access List <ip-access-list>` for
+additional security.
+
+Call a Data API Endpoint
+------------------------
+
+You can call a Data API endpoint from any standard HTTP client. Each
+request can include configuration headers and arguments in the request
+body.
+
+Specify the Request Data Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Data API requests must include a ``Content-Type`` header to specify the
+:ref:`data format <data-api-data-formats>` used in the request body.
+
+- Use ``Content-Type: application/json`` to represent standard JSON types in a
+  Data API request body.
+
+- Use ``Content-Type: application/ejson`` to represent standard JSON
+  types *and* additional EJSON types in a Data API request body.
+
+.. tabs::
+   
+   .. tab:: EJSON
+      :tabid: ejson
+      
+      .. io-code-block::
+         
+         .. input::
+            :language: bash
+            :emphasize-lines: 4
+            
+            curl -X GET \
+              https://data.mongodb-api.com/app/myapp-abcde/endpoint/data/v1/action/insertOne \
+              -H 'api-key: <API Key>' \
+              -H 'Content-Type: application/ejson' \
+              --data-raw '{
+                  "dataSource": "mongodb-atlas",
+                  "database": "learn-data-api",
+                  "collection": "hello",
+                  "document": {
+                    "greeting": "Hello, EJSON!"
+                    "date": { "$date": { "$numberLong": "1654589430998" } }
+                  }
+              }'
+         
+         .. output::
+            :language: json
+            
+            { "insertedId": "629971c0d71aad65bd59c595" }
+   
+   .. tab:: JSON
+      :tabid: json
+      
+      .. io-code-block::
+         
+         .. input::
+            :language: bash
+            :emphasize-lines: 4
+            
+            curl -X POST \
+              https://data.mongodb-api.com/app/myapp-abcde/endpoint/data/v1/action/updateOne \
+              -H 'api-key: <API Key>' \
+              -H 'Content-Type: application/json' \
+              --data-raw '{
+                 "dataSource": "mongodb-atlas",
+                 "database": "learn-data-api",
+                 "collection": "hello",
+                 "filter": { "greeting": "Hello, world!" },
+                 "update": {
+                     "$set": { "greeting": "Hello, universe!" }
+                 }
+              }'
+         
+         .. output::
+            :language: json
+            :visible: false
+            
+            { "matchedCount": 1, "modifiedCount": 1 }
+
+Choose a Response Data Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A request can include an ``Accept`` header to request a specific data
+format for the response body, either JSON or EJSON. If a request does
+not include a valid ``Accept`` header, the response uses the data format
+specified in your Data API configuration.
```

Comment:
consider linking "Data API configuration" to section above (https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/data-api/data-api/generated-endpoints/#response-type) . think this is big enough page to justify using anchor links